tests/baselines/reference/leaveOptionalParameterAsWritten.js

```diff
@@ -32,6 +32,4 @@ declare global {
     }
 }
 //// [c.d.ts]
-declare type Foo = teams.calling.Foo;
-export declare const bar: (p?: Foo) => void;

```

Comment:
This is a case that I'm pretty sure only worked because of a fluke. It's desirable, but when we get the type from the annotation, we get the interface type `Foo` in another file (the symbol is correctly resolved), but then this gets handed down and the info about it being an alias is completely lost. I think this only worked due to assumptions about the enclosing declaration that now no longer hold.